eth_client/lib/eth_client.ex

```diff
@@ -89,6 +89,10 @@ defmodule EthClient do
     wei_to_ether(balance)
   end
 
+  def invoke(method, arguments) do
+    invoke(method, arguments, 0)
+  end
+
   def invoke(method, arguments, amount) do
```

Comment:
Then you can use one function instead:
```elixir
def invoke(method, arguments, amount \\ 0) do
   # .... rest of the function
end
```

And the compiler create the function for you. 

**References:**

* [Default arguments](https://elixir-lang.org/getting-started/modules-and-functions.html#default-arguments)